Editor/AGS.Types/CustomProperties.cs

```diff
@@ -1,16 +1,25 @@
 using System;
 using System.Collections.Generic;
+using System.ComponentModel;
+using System.Runtime.InteropServices;

```

Comment:
It looks like System.Runtime.InteropServices is not needed here. I paid attention because it's a specific namespace for marshalling memory data.

Reply:
must be some left over, removed